### ISSUE TITLE
noDefaultGuild

### ISSUE THREAD

**Issue by DarkArtek** [IrealiTY/crc-bot#8]
Hi and thanks for the bot. i am getting the message noDefaultGuild. Tried to type the fc name and the url with no luck. Could you explain how to set it? thank you

**Comment by GeekyDeaks** [comment]
Hi!  

The bot expects it to be set to the server name as displayed in the server settings page.  You should get a message logged to the console like this 'setting default guild to:'  if it's working correctly.  If you see a message like this 'unable to find default server:' then you might want to check that the name is exactly the same, including case.  If it looks ok, then the only other thing I can suggest is to double check it appears in the member list of the server settings.

**Comment by DarkArtek** [comment]
Awesome! The default guild it's the server. 
Thank You.

**Comment by GeekyDeaks** [comment]
Yeah, it's not very intuitive :D - internally, discord refer to servers as guilds.  Probably should have just had the config as defaultServer!

**Comment by DarkArtek** [comment]
No problem. But i agree that its not very intuitive xD
